### PR TITLE
Use a specific model for word2vec

### PR DESCRIPTION
**Pull request by davenquinn**
I modified the word2vec component to use models out of a "ranked list" constructed from the `/models` endpoint.

Working on allowing model selection...

### CODE REVIEW

**Comment by davenquinn** [comment]
OK, now it has model selection. BONUS: it is refactored to use a separate API context for the Word2Vec API, so we can define that at the top level and not tow a config value all through the application.

File: frontend-shared/src/related-terms/index.ts [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+  const params = {
+    word: word.replace(" ", "_"),
+    model,
+    ...baseParams,
+  };
+
+  const res =
+    useAPIResult("most_similar", params, { context: Word2VecAPIContext }) ?? [];
+  const { buildURL } = useAPIHelpers();
+  const url = buildURL("most_similar", {
```
Comment (iross) [inline frontend-shared/src/related-terms/index.ts:64]: Looks like this isn't building the URL correctly. It ends up linking out to:

https://xdddev.chtc.io/sets/xdd-covid-19/cosmos/apimost_similar?api_key=<REDECATED>&word=test&model=trigram_cleaned&n=50

When it should link to 

https://xdddev.chtc.io/sets/xdd-covid-19/word2vec/api/most_similar?word=test&model=trigram_cleaned&n=50


The actual call to the word2vec API works fine (and correctly doesn't appear to pass along the API key). Is the `useAPIHelpers()` call using a COSMOS context?

**Review by iross — COMMENTED** [review]
This looks great, except the issue re: the "JSON" link out to the word2vec API call, which appears to mix the word2vec and COSMOS API contexts.

File: frontend-shared/src/related-terms/index.ts (hunk 1 above)
Comment (davenquinn) [reply frontend-shared/src/related-terms/index.ts:64]: Nice catch. Fix is exactly as you suggest.

**Comment by iross** [comment]
Nice! Merging and deploying

**Pull request merged**